sources/packages/backend/apps/db-migrations/src/sql/StudentLoanBalances/Create-student-loan-balances.sql

```diff
@@ -0,0 +1,34 @@
+CREATE TABLE sims.student_loan_balances(
+  id SERIAL PRIMARY KEY,
+  student_id INT NOT NULL REFERENCES sims.students(id),
+  csl_balance NUMERIC(8, 2) NOT NULL,
+  balance_date TIMESTAMP WITH TIME ZONE,
```

Comment:
Does it contain the date and time information? If not it must be a date-only column.

Reply:
only date information, let me change it.